.github/scripts/build-binutils.sh

```diff
@@ -6,6 +6,8 @@ TARGET=${TARGET:-aarch64-w64-mingw32}
 BUILD_PATH=${BUILD_PATH:-$PWD/build-$TARGET}
 BUILD_MAKE_OPTIONS=-j$(nproc)
 INSTALL_PATH=${INSTALL_PATH:-~/cross}
+RUN_CONFIG=${RUN_CONFIG:-1}
```

Comment:
Why not zero?

Reply:
Please elaborate your thoughts. I'd expect that one would expect full clean build by default.

Reply:
please nvm, misreading as -1 )